docker-compose.yaml

```diff
@@ -103,6 +103,22 @@ services:
       - POSTGRES_DATABASE=postgres
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=${POSTGRES_ADMIN_PASSWORD-postgres}
+  gptscript_runner:
+    build:
```

Comment:
let's use image here, not build - if you use :latest, it gets automatically subbed out for a pinned version in the CI build scripts